spec/presenters/openid_connect_certs_presenter_spec.rb

```diff
@@ -4,17 +4,23 @@
   subject(:presenter) { OpenidConnectCertsPresenter.new }
 
   describe '#certs' do
-    it 'renders the server public key as a JWK set' do
+    it 'renders the server public keys as a JWK set' do
       json = presenter.certs
 
-      expect(json[:keys].size).to eq(1)
+      expect(json[:keys].size).to eq(2)
       expect(json[:keys].all? { |k| k[:alg] == 'RS256' }).to eq(true)
       expect(json[:keys].all? { |k| k[:use] == 'sig' }).to eq(true)
 
-      key_from_response = JWT::JWK.import(json[:keys].first).public_key
-      public_key = AppArtifacts.store.oidc_public_key
+      # Primary key should be first
+      primary_key_from_response = JWT::JWK.import(json[:keys].first).public_key
```

Comment:
The `.keys[1]` jumped out at me, maybe we could destructuring to avoid hardcoding indexes?

```suggestion
      primary_key_from_response, secondary_key_from_response = json[:keys].map do |key|
        JWT::JWK.import(key).public_key
      end
```